### ISSUE TITLE
[Bug]: Digital Miner (Mekanism), which I installed in End, crashes every time I open the game.

### ISSUE THREAD

**Issue by ramazansancar** [AllTheMods/All-the-mods-9-Sky#507]
### Possible Fixes

Yes

### Modpack Version

1.0.7

### What happened?

Every time I open the Digital Miner game that I installed in End, the Flux Points and Ender Chest that I gave energy to are broken and destroyed. 2. It is broken in Miner and is in the digital miner inventory.

![image](https://github.com/AllTheMods/All-the-mods-9-Sky/assets/8878055/1bd8dec0-f509-4759-9afa-06acdee30db5)

[latest.log](https://github.com/user-attachments/files/16110699/latest.log)

### latest.log

https://github.com/user-attachments/files/16110699/latest.log

**Comment by ramazansancar** [comment]
Additional info:
Those who go to the inventory go to the me system with the connected ender chest. It doesn't disappear.

![image](https://github.com/AllTheMods/All-the-mods-9-Sky/assets/8878055/e6425706-2d9a-4361-be96-12aabd668ae6)

**Comment by ramazansancar** [comment]
Is there any progress?

**Comment by ramazansancar** [comment]
> Is there any progress?

Is there any progress?

**Comment by Jmpp2909** [comment]
Check if you accidentally configured the digital miner to mine at it's own level, if so, its destroying everything including itself.
If not, ill need a proper crash log, found in the crash-reports folder
Or the latest.log of when the game crashes, before opening it again.

**Comment by ramazansancar** [comment]
> Check if you accidentally configured the digital miner to mine at it's own level, if so, its destroying everything including itself. If not, ill need a proper crash log, found in the crash-reports folder Or the latest.log of when the game crashes, before opening it again.

It's been a year since I posted the issue, so I'm not sure exactly what's going on. However, I'll be posting any new findings I find. @Jmpp2909

**Comment by Jmpp2909** [comment]
Fair enough, since i didnt see any other case like this, ill be closing this issue, reopen it if it happens again in a new version
